pvlib/irradiance.py

```diff
@@ -994,6 +994,56 @@ def king(surface_tilt, dhi, ghi, solar_zenith):
     return sky_diffuse
 
 
+def muneer(surface_tilt, dhi, b):
+    '''
+    Determine diffuse irradiance from the sky on a tilted surface using
+    the Muneer [1]_ model.
+    
+     Parameters
+    ----------
+    surface_tilt : numeric
+        Surface tilt angle in decimal degrees. Tilt must be >=0 and
+        <=180. The tilt angle is defined as degrees from horizontal
+        (e.g. surface facing up = 0, surface facing horizon = 90)
+
+    dhi : numeric
+        Diffuse horizontal irradiance in W/m^2. DHI must be >=0.
+    
+    b : numeric
+        Radiance distribution index, introduced by Moon and Spencer [2]_ to model
+        luminance distribution of overcast sky.
+        
+        'best' values of b as found for Easthampstead data:
+           isotropic: b = 0
+           shaded surface: b = 5.73
+           sunlit surface under overcast sky: b = 1.68
+           sunlit surface under non-overcast sky: b = -0.62
+        
```

Comment:
```suggestion
        Recommend values from [X]_:

           - isotropic: b = 0
           - shaded surface: b = 5.73
           - sunlit surface, overcast sky: b = 1.68
           - sunlit surface, non-overcast sky: b = -0.62
```
What is the correct reference for these values?